### DISCUSSION TITLE
Can't make Executable with customtkinter

### DISCUSSION THREAD

**Discussion by Sayad-Uddin-Tahsin** [TomSchimansky/CustomTkinter#1984] [Q&A]
I've been trying to make an EXE with customtkinter, seems the EXE is not properly functioning!

### Pyinstaller command: 
```shell
pyinstaller --name="IPCom" --icon=./Assets/IPCom.ico --version-file=IPCom-Version.txt --onedir --windowed --add-data "G:/Python Projects/IPCom/.venv/Lib/site-packages/customtkinter;customtkinter/" ipcom_gui.py
```
### Error:
![image](https://github.com/TomSchimansky/CustomTkinter/assets/89304780/e3f0f229-ecc2-4b8b-8d88-7fcdb09d146c)
 
According to the [Documentation](https://customtkinter.tomschimansky.com/documentation/packaging), what did I do wrong?

**Comment by AlbertSolomon** [comment]
@Sayad-Uddin-Tahsin In my experience, I found working with the [cx-freeze library](https://cx-freeze.readthedocs.io/en/stable/) to be easier to use. While cx-freeze focuses on building your application, you'll need to package it separately using  [inno-setup](https://jrsoftware.org/isdl.php).  It might not be what you wanted for an answer, but i felt like providing you with an extra option in case if using `pyinstaller` doesn't work out.

**Comment by Akascape** [comment]
@Sayad-Uddin-Tahsin 
Duplicate of  https://github.com/TomSchimansky/CustomTkinter/issues/1829 

For a detailed guide related to pyinstaller, check here: https://github.com/TomSchimansky/CustomTkinter/discussions/939
